.devcontainer/Dockerfile

```diff
@@ -5,7 +5,7 @@ RUN dnf -y install \
       tar \
       gzip \
       git \
-      dotnet-sdk-8.0 \
```

Comment:
Did this help with the vs code stuff by chance or nah?

Reply:
No I've just been editing the backend in visual studio